### ISSUE TITLE
Time Travel

### ISSUE THREAD

**Issue by Rocks4Free** [yogstation13/yogstation#2048]
Would be a traitor item for 20 TC, would be usable once per round

When used would allow the user to go back a specific amount of time (with a limit)

When time goes back, everyone else would either maintain control or would be forced to do exactly what they did before unless interracted with by the time traveller/something else different

Would cause mass hysteria

/shitpost

**Comment by ShadowDeath6** [comment]
no

**Comment by Rocks4Free** [comment]
yes

**Comment by 00aquir** [comment]
how about jumping back in the place the traitor was X minutes ago (in time), for heavy amount of TC, since it could be hard to interpret where he was exactly, and even if he gets out of a bad situation or something, he would still be wanted by sec (for example), and even before this he would be left with only a small amount of TC or a Predetermined item choice (which would prohibit from getting the easy-to-use items) which would make this possible to be balanced?

**Comment by Rocks4Free** [comment]
Actually this would probably be better as a wizard spell or (god forbid) a lavaland treasure

**Comment by 00aquir** [comment]
The wizard has teleport and it might be balanceable to be both a miner's (recording positions every minute from the point he found it) and a traitor item?

**Comment by Mrcreepssss** [comment]
getting a timetravel thing could be good, but NOT LIKE THIS!

perhaps that it sends you to a backup station where all is how it was that time ago and you can steal already stolen things? if they get zapped away from this one is another thing entirely. and so is if there's people and how they can interact with ye.

**Comment by Alek2ander** [comment]
Yeah, keeping track of ALL THE STUFF so you can revert and shit? Not gonna happen. There are hundreds of thousands of objects to keep track of.

**Comment by Alek2ander** [comment]
That said, something else with a time travel flavor could be a reasonable thing.
For example, a time machine can temporarily duplicate items (by providing you with a version of the item from the future) but if the past version is not sent back in time to complete the time loop, something horrible happens.
For supreme memes, summon your own future self, controlled by a ghost (but then, of course, you will need to deposit yourself).

**Comment by Frostben** [comment]
Why not make it sorta like tracers time abilities from over watch? That seems to be much more manageable. You would activate the time device and it would teleport you back to the location you were in x minutes ago with the same health and stats from x minutes ago

**Comment by Rocks4Free** [comment]
> That said, something else with a time travel flavor could be a reasonable thing.
For example, a time machine can temporarily duplicate items (by providing you with a version of the item from the future) but if the past version is not sent back in time to complete the time loop, something horrible happens.
For supreme memes, summon your own future self, controlled by a ghost (but then, of course, you will need to deposit yourself).

This actually sounds cool. You would use the spell/item/whatever to duplicate yourself on the spot through time travel, and your duplicate is controlled by a ghost. They would have your objectives, items (when you "summon" them) and they would have to obey you. You would have to send yourself back in time later, probably within a certain time limit, and the original player would be transferred to the duplicate. The ghost  would just be evicted. (rip) Or it could go the other way, and you'd just be nonexistant for a little while.

**Comment by Kmc2000** [comment]
Have it take you back to a victorian/ other period in history styled room with loads of storage space, basically a fluff storage area which you can jump to and fro at a heavy power cost, ie you need to hook up this device to the powernet?

**Comment by Super3222** [comment]
I've always had time travel in mind. I'll try to find my idea for a mini-antagonist revolved around time travel.

A tracer ability could also work somewhat. Maybe add an option to make drop points manual (where you set it by clicking an action button) or automatic (where it sets drop points every 5 seconds), and another option button where the user can travel back to that drop point. I don't think keeping track of every item on them is a good idea. Keeping track of health is guaranteed.

**Comment by doerakpoes** [comment]
Quickly alter the chrono suit!

**Comment by Mrcreepssss** [comment]
"This actually sounds cool. You would use the spell/item/whatever to duplicate yourself on the spot through time travel, and your duplicate is controlled by a ghost. They would have your objectives, items (when you "summon" them) and they would have to obey you. You would have to send yourself back in time later, probably within a certain time limit, and the original player would be transferred to the duplicate. The ghost would just be evicted. (rip) Or it could go the other way, and you'd just be nonexistant for a little while."

maybe it'd be better if you get transferred to the future self and the ghost becomes the current self? you'd have to guard the guy though, if he dies you should get chrono erased untill your past self gets rezzed then you can come back aswell. perhaps you could even make it so pastguy's damage taken is done onto the futureself? then we perhaps shouldn't give it a time limit. but if we were to do this why not make it into a new antag all in it's own?

**Comment by Mrcreepssss** [comment]
then we could make it so they'd have to chrono-erase people to get chrono-clones of themselves.
